src/adv.py

```diff
@@ -49,3 +54,96 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+player = Player("JoeBob", room["outside"])
+
+
+
+# *********************************************************************
+# *** Start the game loop and take care of printing all neccessary info
+# *********************************************************************
+
+print("Welcome to Adventure Quest!")
+
+gameOver = False
+
+while not gameOver:
+
+  print("")
+  print (f"Current room: {player.currentRoom.name}")
+  print (f"Current room: {player.currentRoom.description}")
+
+
+  # *** Print out the items in the room
+
+  itemStr = "The room contains the following items: "
+
+  if len(player.currentRoom.items) > 0:
+    for item in player.currentRoom.items:
+      # *** On the last item in the array, output special formatting
+      if item == player.currentRoom.items[-1]:
+        itemStr = itemStr + item.name
+      else:
+        itemStr = itemStr + item.name + ", "
+
+    print(itemStr)
+
+
+
+  # **********************************
+  # *** Get and make the player's move
+  # **********************************
+
+  move = None
+  inputStr = input("Please indicate which direction you'd like to move, either n, e, s, or w, q to quit: ")
+  split = inputStr.split()
+
+  # *** If the length of the split array is two, maybe there's a command and an item
+  if len(split) == 2:
+
+    # *** Get or take an item
+    if split[0] == "take" or split[0] == "get":
+      player.get_item(split[1])
+
+    # *** Drop an item
+    elif split[0] == "drop":
+      player.drop_item(split[1])
+
+  # *** If the length of the split array is one, probably just tryna move somewhere or quit or look at the inventory
+  elif len(split) == 1:
+    move = split[0]
+
+    if move == "q":
+      gameOver = True
+
+    elif move == "i" or move == "inventory":
+      print("Your inventory:")
+      for item in player.items:
+        print(item.name + "\n" + item.description)
+
+    elif move == "n":
+      if player.currentRoom.n_to == None:
+        print("No room to the North")
+      else:
+        player.currentRoom = player.currentRoom.n_to
+
+    elif move == "e":
+      if player.currentRoom.e_to == None:
+        print("No room to the East")
+      else:
+        player.currentRoom = player.currentRoom.e_to
+
+    elif move == "s":
+      if player.currentRoom.s_to == None:
+        print("No room to the South")
+      else:
+        player.currentRoom = player.currentRoom.s_to
+
+    elif move == "w":
+      if player.currentRoom.w_to == None:
+        print("No room to the West")
+      else:
+        player.currentRoom = player.currentRoom.w_to
+
```

Comment:
The commands for n, s, w, and e could be simplified by creating a move method in the Player class.